probatus/feature_elimination/feature_elimination.py

```diff
@@ -673,6 +685,37 @@ def get_reduced_features_set(self, num_features):
         else:
             return self.report_df[self.report_df.num_features == num_features]["features_set"].values[0]
 
+    def get_reduced_features_set_shap_values(self, num_features, abs_shap_values=True):
```

Comment:
Maybe good to rename it to `get_feature_importance`?

Reply:
Some small unit test using very very simple handcrafted dataset, to get these shap values would be nice.